Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -695,9 +695,29 @@ def test_log
         system "git", "commit", "--allow-empty", "-m", "This is a test commit"
       end
     end
+
+    core_tap = CoreTap.new
+    formula_file = core_tap.formula_dir/"testball.rb"
+    formula_file.write <<-EOS.undent
+      class Testball < Formula
+        url "https://example.com/testball-0.1.tar.gz"
+      end
+    EOS
+
+    FileUtils.cd "#{core_tap.path}".chomp do
```

Comment:
The `.chomp` doesn't look like it belongs here. Better yet, use the [`Pathname#cd` method](https://github.com/Homebrew/brew/blob/21ca138edf584daebf1795a52d6edd6ec34ce605/Library/Homebrew/extend/pathname.rb#L327-L329), i.e. `core_tap.path.cd do`, instead of `FileUtils.cd`.